security/src/main/java/uk/ac/ox/ndph/mts/security/authorisation/AuthorisationService.java

```diff
@@ -99,20 +100,30 @@ public boolean authorise(String requiredPermission) {
      */
     public boolean authorise(String requiredPermission, List<String> entitiesSiteIds)  {
 
+
         try {
             //Get the user's object id
             String userId = securityContextUtil.getUserId();
             String token = securityContextUtil.getToken();
 
-            LOGGER.debug("userId is - " + userId);
-            LOGGER.debug("managed identity is - " + managedIdentity);
+            LOGGER.debug("userId Is - {}", userId);
+            LOGGER.debug("managed identity is - {}", managedIdentity);
```

Comment:
Debug code - remove unless wanted for debugging in QA

Reply:
We discussed in a different thread and decided to leave it in as debugging for now while the managed service identity is still being worked on (not in this story).